### DISCUSSION TITLE
@Mike777Khoo 付款1个星期了，还没收到验证码！！！！！

### DISCUSSION THREAD

**Discussion by TangAssos** [getlantern/lantern-forum-cn#618] [Q&A]
@Mike777Khoo 购买lantern已经好几天了，到现在也没收到激活码，也没任何反馈。我付款买东西，东西没买到，赶紧退款，邮箱：

**Comment by MaxMKMax** [comment]
@TangAssos 您好. 我这里帮您提交处理问题. 请给我们 2 天时间. 保证帮您弄好. 谢谢您的耐心等待.

如果 2 天之后无法帮您弄好的话, 我帮您申请退款. 请您放心.

**Reply by TangAssos** [reply]
@Mike777Khoo 就2天，48小时，我跟帖等你回复

**Reply by MaxMKMax** [reply]
@TangAssos 好的. 没问题. 给我们 48 个小时. 谢谢您.

**Reply by MaxMKMax** [reply]
@TangAssos 
来了来了. 答应您的做到了! 这是您专属的激活码. 24 小时内给您解决了. 谢谢您的耐心等待. 给您点赞!
HBWRP-VD9CM-TKR4F-VKC87-7WYVM

**Reply by JW2006778** [reply]
楼主的问题24小时就解决了，我14号支付至今一直在拖，是不是只有提出退款才给解决？

**Reply by MaxMKMax** [reply]
@JW2006778 您好. 不是的. 可能是客服看漏了. 要不您这里提交给我您的支付单据和账号邮箱吧. 我手动帮您提交提交问题. 还有您是遇到什么问题呢?

以后最好是自己开个帖子. 很多人在别人的帖子底下留言, 也很容易看漏的. 麻烦您了. 谢谢您!

**Reply by JW2006778** [reply]
我开了帖子的、也多次提交了凭证。
我真不明白，支付时都有email地址，系统直接给出激活码或验证码即可；就算有漏洞再次提交凭证后系统给出激活码或验证码，这在几十年前就实现了的东西，为什么现在要靠仅有的一个人来做middleware？
